.github/workflows/deleted-files.yml

```diff
@@ -0,0 +1,38 @@
+name: Deleted files
+
+on:
+  pull_request:
+    branches:
+      master
+
+jobs:
+  list-deleted-files:
+    runs-on: ubuntu-latest
+    name: Detect deleted files
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Get changed docs md files
+        id: changed-files
+        uses: tj-actions/changed-files@v37
+        with:
+          files: docs/**/*.md
+      - name: Format list of deleted files
+        id: format-output
+        env:
+          ORIGINAL: '${{ steps.changed-files.outputs.deleted_files }}'
+        run: |
+          echo "FORMATTED=$(echo $ORIGINAL | sed 's/ /`<br>`/g')" >> $GITHUB_OUTPUT
+      - name: Comment PR if there are deleted files
+        if: steps.changed-files.outputs.any_deleted == 'true'
+        uses: thollander/actions-comment-pull-request@v2
+        with:
+          message: |
+            ## Notice
+            
+            This PR deletes one or more `md` files:
+            `${{ steps.format-output.outputs.FORMATTED }}`
+            
+            Remember to append the appropriate redirects in `mkdocs.yml` under `plugins.redirects.redirect_maps`.
+          comment_tag: deleted-files-reminder
```

Comment:
FYI, this line causes the action to overwrite the comment, so we don't get spammed.